src/ducks/payments/api.js

```diff
@@ -0,0 +1,67 @@
+import { getBiPayment } from './storage'
+
+async function handleErrors(response) {
+  if (!response.ok) {
+    const error = await response.json()
+    throw {
+      status: `${response.status} - ${response.statusText}`,
+      message: error
+    }
+  }
+  const data = await response.json()
+  return data
+}
+
+export const getPayment = () => {
+  const biPayment = getBiPayment()
+  return biPayment
+}
+
+const headers = () => {
+  const h = new Headers()
+  h.append('Accept', 'application/json')
+  h.append('Content-Type', 'application/json')
+
+  const biPayment = getBiPayment()
+  const token = biPayment && biPayment.token
+  if (token) {
+    h.append('Authorization', `Bearer ${token}`)
+  }
+  return h
+}
+
+export const biApi = (method, url, payload) => {
+  const config = { method, headers: headers() }
+  if (payload) {
+    config.body = JSON.stringify(payload)
+  }
+  return fetch(new Request(url, config)).then(handleErrors)
```

Comment:
handleErrors does a bit more than handlingError, it also takes care of parsing as a JSON, I think we should find a more generic name, `handleBIResponse` ?

I am not sure that it should be extracted into another method, it looks like it would be simpler to just await the fetch here, and do the `handleErrors` logic directly here.